### PR TITLE
css-three-column-layout - Ariel Donner

### PR DESCRIPTION
**Pull request by arieldonner**
css-three-column

### CODE REVIEW

File: css-three-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  justify-content: center;
+}
+
+.column-full {
+  width: 100%;
+}
+
+.column-one-third {
+  width: calc(100% / 3);
+  flex-basis: 30%;
```
Comment (Zhiwutian) [inline css-three-column-layout/styles.css:22]: if you are going to use `width` I would remove the flex basis here.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Just one tiny update. Very nice work!

File: css-three-column-layout/styles.css (hunk 1 above)
Comment (arieldonner) [reply css-three-column-layout/styles.css]: css-three-column

**Review by Zhiwutian — APPROVED** [review]
Very nice!

**Pull request merged**